module-2/backend/week_one.md

```diff
@@ -1,34 +1,80 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
+1.List the five common HTTP verbs and what the purpose is of each verb.
+  *Get-Is used to return data from the database. It is used to render a view.
+  *Post-Is used to create a database entry.
+  *Put-Is used to update an existing database entry.
+  *Patch-Is used to update specific attributes of a database entry.
+  *Delete-Is used to delete a database entry.
 2. What is Sinatra?
+  *A framework for ruby.
 4. What is MVC?
+  *Model, View, Controller. It's a pattern for creating an app.
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  *So that we can be replaced by other devs. Really so that our code is readable to others, and probably to us.
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  *Instance variables.
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
-  
+
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+
+```ruby
+get '/horses' do
+  erb :index #{:name => 'Mr. Ed'}
+end
+```
+
 9. What's the purpose of ERB?
+  * To be able to interpolate ruby code into HTML
 10. Why do I need a development AND test database?
+  * So you can control the data in the test database and know what your conditions are.

```

Comment:
And so the data from your test database and development database dont interfere.